shotover-proxy/example-configs/redis-cluster-hiding/docker-compose.yml

```diff
@@ -0,0 +1,90 @@
+version: '2'
```

Comment:
I know you didn't write this whole file, but any idea why this is '2' while the others are '3.3'

Reply:
yeah its a bit all over the place.
I think its out of scope this PR but you could raise an issue to unify onto a single version if you wanted.